apps/ui-kit/src/lib/styles/index.css

```
@@ -12,7 +12,7 @@
 @import 'tailwindcss/utilities';
 @import './utilities.css';
 
-:root.light,
+:root,
 :root.dark {
```

This could be simplified to just this I guess then right?
```suggestion
:root {
```